src/main/g8/scalafix/build.sbt

```diff
@@ -1,11 +1,15 @@
 lazy val V = _root_.scalafix.sbt.BuildInfo
+
+lazy val rulesCrossVersions = Seq(V.scala213, V.scala212, V.scala211)
+lazy val scala3Version = "3.0.0"
```

Comment:
As 3.x releases come out, we might want to test against different versions (although upgrading scala should not be as hard as on the 2.x era obviously). When/if that happens, we should use the full scala version as matrix suffixes, but I don't see a reason to do it now.